core/src/main/java/org/apache/brooklyn/core/catalog/internal/BasicBrooklynCatalog.java

```diff
@@ -566,6 +575,7 @@ private void collectCatalogItems(String sourceYaml, Map<?,?> itemMetadata, List<
             .deprecated(catalogDeprecated)
             .iconUrl(catalogIconUrl)
             .plan(sourcePlanYaml)
+            .referenceType(referenceType)
```

Comment:
What if the type is another catalog item, my understanding is that this must be a class name?